### PR TITLE
Add feature image gallery.

### LINKED ISSUE #7: Week 2 Project - 4: Image Gallery
Add images! Create a gallery of your favorite places, or selfies, or pictures of your pets.

### PR DESCRIPTION
**Pull request by SibingWu**
## Description

Add images! Create a gallery of your favorite places, or selfies, or pictures of your pets.
Allow the user to go back to the previous page from gallery page.

Fixes #7 
Files Changed: index.html; script.js; images; images.html

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: portfolio/src/main/webapp/images.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        <meta charset="UTF-8">
+        <title>Sibing's image gallery</title>
+        <link rel="stylesheet" href="style.css">
+        <script src="script.js"></script>
+    </head>
+    <body>
+        <div id="content">
+            <h1>Sibing's Image Gallery</h1>
+            <div id="gallery">
+                <a href="images/Sibing.jpg"><img src="images/Sibing.jpg" style="height:150px; width:200px"></a>
```
Comment (htthach) [inline portfolio/src/main/webapp/images.html]: Look like each image has a different naming convention snake, camel, etc. Should we choose one? anything is fine.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -26,3 +26,16 @@ function addRandomGreeting() {
...
+ * Goes back to the previous page.
+ */
+function goBack() {
+    //if it was the first page
+    if(history.length === 1){
+        window.location = "index.html"
+    } else {
+        history.back();
+    }
+    // window.history.back();
```
Comment (htthach) [inline portfolio/src/main/webapp/script.js]: this comment can be removed.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -26,3 +26,16 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Goes back to the previous page.
+ */
+function goBack() {
+    //if it was the first page
```
Comment (htthach) [inline portfolio/src/main/webapp/script.js]: Maybe replace "was" with "is"? Keep simple present tense unless it's clearly past tense.
And need a space and capital letter. So maybe:
// If this is the first page.

**Review by htthach — COMMENTED** [review]
Wonder if the walk through guide mentions writing tests?

File: portfolio/src/main/webapp/images.html (hunk 1 above)
Comment (SibingWu) [reply portfolio/src/main/webapp/images.html]: Fixed

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (SibingWu) [reply portfolio/src/main/webapp/script.js]: Fixed

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (SibingWu) [reply portfolio/src/main/webapp/script.js]: Fixed

**Review by htthach — APPROVED** [review]

**Pull request merged**